module-system/module-implementations/sov-chain-state/src/lib.rs

```diff
@@ -3,7 +3,6 @@
 
 /// Contains the call methods used by the module
 mod call;
-pub use call::*;
```

Comment:
I think the call implementations of the module will be required downstream. Any particular reason we are removing this re-export? Note that the `call` module is private.